tests/testthat/tests-clone_instat_calculation.R

```diff
@@ -1,27 +1,5 @@
-library(testthat)
-library(R6)
```

Comment:
We do not need to read in `testthat` each time

In addition, when you load in `R6` is when you create mocked/cloned objects. So, we shouldn't read this in either